mkosi.postinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+mkosi-chroot \
+    useradd \
+    --user-group \
+    --create-home \
+    --password "$(openssl passwd -1 testuser)" \
+    --groups sudo,systemd-journal \
+    --shell /bin/bash \
+    testuser
```

Comment:
Let's be clear on where this comes from
```suggestion
    mkosiuser
```

Reply:
I don't think it matters since it's the default image, this is intended for use by developers, not users

Reply:
But we can name the user mkosi to make it shorter I guess